content/4_Gold/Tree_Euler.mdx

```diff
@@ -132,7 +130,7 @@ If we can preprocess a rooted tree such that every subtree corresponds to a cont
 
 ### Implementation
 
-After running `dfs()`, each range `[st[i], en[i]]` contains all ranges `[st[j], en[j]]` for each `j` in the subtree of `i`. Also, `en[i]-st[i]+1` is equal to the subtree size of `i`.
+After running $\text{dfs}()$, each range `[st[i], en[i]]` contains all ranges `[st[j], en[j]]` for each `j` in the subtree of `i`. Also, $\texttt{en}[i]-st[i]+1$ is equal to the subtree size of `i`.
```

Comment:
You forgot \texttt for the second st[i]. Also, maybe use latex instead of code for [st[i], en[i]] for consistency?

Reply:
good idea

Reply:
texified the node numbers too (i think there is clear enough distinction but maybe we want to make it more emphasized?)

Reply:
Looks good. Can you also just add "node" to things like "because 2 has no children" to make it "because **node** 2 has no children" please?